### ISSUE TITLE
Color of icons not visible

### ISSUE THREAD

**Issue by SenneVanderAuwera** [SenneVanderAuwera/startpage#37]
The default color of the icon font is not set.

**Comment by KrisVda1** [comment]
I agree. And all the other colors are set to black. So i recommend a standard scheme like the old colours to be set as default.

**Comment by SenneVanderAuwera** [comment]
e870128aeac53582e0925fec767feba52c31bdbe
